services/QuillLMS/app/models/question.rb

```diff
@@ -52,8 +52,9 @@ class Question < ApplicationRecord
     TYPE_GRAMMAR_QUESTION => 'grammar_questions',
   }
 
-  INCORRECT_SEQUENCES = 'incorrectSequences'
-  FOCUS_POINTS = 'focusPoints'
+  INCORRECT_SEQUENCES = TranslatableQuestion::INCORRECT_SEQUENCES
+  FOCUS_POINTS = TranslatableQuestion::FOCUS_POINTS
+  CMS_RESPONSES = TranslatableQuestion::CMS_RESPONSES
```

Comment:
I imagine that you have these here because something doesn't work without them, but I would have expected these constants to be set via the inclusion of `TranslatableQuestion`.  Weird that you have to declare them again.

Reply:
They are set, but I thought it looked nicer to not have to prefix them since they're used so much in both files. That said, CMS_RESPONSES is not needed here.